web/views.py

```
@@ -626,14 +628,14 @@ def course_search(request):
     if query:
         courses = courses.filter(
             Q(title__icontains=query)
-            | Q(description__icontains=query)
+            | Q(description__icontains(query))
```

Ruff (0.8.2)

Undefined name `description__icontains`

(F821)